src/content/1.4/code/ocaml/snippet06.ml

```diff
@@ -1,3 +1,6 @@
-let to_words : string -> string list writer =
- fun s -> String.split s ~on:' ', "to_words "
-;;
+(* val up_case : string -> string writer
+   Note: OCaml strings are raw bytestrings, not UTF-8 encoded. *)
```

Comment:
The book never mentions UTF-8 encoding.. Maybe this makes sense in an OCaml tutorial to note, but why here?